shared/util/string.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+function toStringForLog(a: any): string {
+  if (typeof a === 'string') {
+    return a
+  } else if (a instanceof Error) {
+    return a.toString()
+  } else if (typeof a === 'object') {
+    return JSON.stringify(a)
+  }
+  return (a.toString && a.toString()) || 'Failed to turn item to string in toStringForLog'
```

Comment:
it seems this should go before the object check, right? otherwise things with `toString` will be `JSON.stringify`ed first.

Reply:
every object has a .toString which returns `[object object]`

Reply:
d'oh, right